tests/design_assistant/processing/test_design_assistant_service.py

```diff
@@ -186,6 +186,7 @@ def mock_get_session_for_property(input):
                                        'design_targets': [{'design_target_name_field': 'Workability',
                                                            'design_target_optimization_field': 'maximize',
                                                            'design_target_value_field': '10 MPa'}],
+                                       'formulation_field': None,
```

Comment:
es muss noch mehr in den Tests angepasst werden. Zum Bsp. schlägt test_update_design_assistant_session_calls_persistence_with_powders fehl. Außerdem müssen die Werte etwa opc etc. auch noch auf die neue Konvention mit Großschreibung etc. angepasst werden.